vm/devices/get/get_protocol/src/lib.rs

```diff
@@ -26,6 +26,8 @@ pub const MAX_HEADER_SIZE: usize = 256;
 // (required due to underlying vmbus pipe message size constraints)
 pub const MAX_PAYLOAD_SIZE: usize = 8192;
 
+pub const HOST_VMWP_ATTESTATION_GENERIC_ERROR_CODE: usize = 0xFFFFFFFF;
```

Comment:
nit: move this constant closer to the packets that use it.

also, you currently slotted it between some const declarations, and a `const_assert!` that relates those constants, which isn't great form

Reply:
Updated.